Cargo.toml

```diff
@@ -59,6 +59,9 @@ tracing = { version = "0.1", features = ["log"] }
 thiserror = "1.0"
 png = "0.17"
 bitflags = "2.4"
+byteorder = "1.5"
+num-derive = "0.4"
+num-traits = "0.2"
```

Comment:
note: This is all crates that we don’t want in the future, so I wouldn’t encourage adding them as workspace dependencies